R/gs_update_ahr.R

```diff
@@ -64,8 +60,9 @@
 #' # Randomization ratio
 #' ratio <- 1
 #'
-#' # Example A: one-sided design (efficacy only) ----
-#'
+#' # ------------------------------------------------- #
+#' # Example A: one-sided design (efficacy only)
+#' # ------------------------------------------------- #
```

Comment:
@nanxstats previously had me standardize all the comments to not use these ASCII art boxes. Please see #308 and #309 for details

Reply:
Hello @jdblischak, I wanted to bring up the issue of heavy examples on this help page. As a maintainer, it becomes challenging for me to have a clear understanding of all the ongoing examples without the aid of art boxes, and this difficulty extends to first-time users as well. I strongly believe that art boxes would greatly assist users in implementing the code more easily. If there isn't a distinct right or wrong solution, could we consider keeping the art boxes?